src/services/pool/decorators/pool.decorator.ts

```diff
@@ -78,14 +110,15 @@ export class PoolDecorator {
    */
   private async getSnapshots(): Promise<Pool[]> {
     const currentBlock = await this.providerService.getBlockNumber();
-    const blockNumber = await getTimeTravelBlock(currentBlock);
+    const blockNumber = getTimeTravelBlock(currentBlock);
     const block = { number: blockNumber };
     const isInPoolIds = { id_in: this.pools.map(pool => pool.id) };
     try {
-      return await this.poolSubgraph.pools.get({
+      const data = await this.poolSubgraph.pools.get({
         where: isInPoolIds,
         block
       });
+      return data;
```

Comment:
Looks like this might have been added for logging? Unnecessary const.

Reply:
yeah, fixed